lib/data_update_scripts/20210108033107_remove_looking_for_work_profile_fields.rb

```diff
@@ -0,0 +1,10 @@
+module DataUpdateScripts
+  class RemoveLookingForWorkProfileFields
+    def run
+      # destroy_by is idempotent by default: if no record can be found an empty
```

Comment:
Note: I did not add a spec for this script as it would essentially just retest `destroy_by` which didn't seem worthwhile.